src/libraries/Directory.Build.targets

```diff
@@ -49,6 +49,7 @@
     <IsNETCoreAppRef Condition="'$(IsNETCoreAppRef)' == '' and
                                 '$(IsReferenceAssembly)' == 'true' and
                                 $(NetCoreAppLibrary.Contains('$(AssemblyName);'))">true</IsNETCoreAppRef>
+    <IsNETCoreAppPackage Condition="$(NetCoreAppLibraryPackage.Contains('$(AssemblyName);'))">true</IsNETCoreAppPackage>
```

Comment:
Let me know what you think of the name here.  @ViktorHofer @Anipik this is what I was suggesting for using with ProjectReference->PrivateAssets.  Not sure if you already have something like this yet.

Reply:
Looks good to me. I wanted to implement it as an item actually and convert the NetCoreAppLibrary property to that but that can happen later as a clean-up as it will touch many more places than necessary here.